phare/pharein/maxwellian_fluid_model.py

```diff
@@ -6,8 +6,14 @@ class MaxwellianFluidModel(object):
 
     def defaulter(self, input, value):
         if input is not None:
-            from inspect import signature
-            assert len(signature(input).parameters) == self.dim
+            import inspect
```

Comment:
I don't get the changes

Reply:
this allows for functions which use (*args), the functors do this to let any dim be used and check the length of the list

Reply:
with reverting the Functors, this becomes unused, but I think it could be useful to keep, unless there's a good reason why we should not allow it of course

Reply:
ok